ansible_base/oauth2_provider/models/id_token.py

```diff
@@ -0,0 +1,19 @@
+import oauth2_provider.models as oauth2_models
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+
+from ansible_base.lib.abstract_models.common import CommonModel
+
+activitystream = object
+if 'ansible_base.activitystream' in settings.INSTALLED_APPS:
+    from ansible_base.activitystream.models import AuditableModel
+
+    activitystream = AuditableModel
+
+
+class OAuth2IDToken(CommonModel, oauth2_models.AbstractIDToken, activitystream):
```

Comment:
Do we need/want prevent search on the `jti` field from AbstractIDToken?
```
    * :attr:`jti` ID token JWT Token ID, to identify an individual token
```

This would also apply to the other tokens as well if we are going to do this.

Reply:
I looked into this when I added `prevent_search()` on the access token and I _think_ the JTI isn't secret? It's just a UUID that identifies a token, but not a token itself. It would be nice to have someone more familiar clarify that though.

Reply:
From a quick scan of the awx codebase it looks like they don't do anything with the jti field so we are fine to leave this as is.